src/renderer/src/routes/pages/BuyAndSellTokens/BuyTokensContent.tsx

```diff
@@ -0,0 +1,99 @@
+import { Dispatch, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { MdRestartAlt } from 'react-icons/md'
+import fingerprint from '@fingerprintjs/fingerprintjs'
+import { GateFiDisplayModeEnum, GateFiEventTypes, GateFiSDK } from '@gatefi/js-sdk'
+import { Button } from '@renderer/components/Button'
+import { UtilsHelper } from '@renderer/helpers/UtilsHelper'
+import { useMountUnsafe } from '@renderer/hooks/useMount'
+import { useCurrencySelector } from '@renderer/hooks/useSettingsSelector'
+import { IAccountState } from '@shared/@types/store'
+import { theme as tailwindTheme } from '@shared/libs/theme'
+
+import { BuyAndSellTokensLayout } from './BuyAndSellTokensLayout'
+import { buyTokensIframeUrl, hideBrand, lang, merchantId, theme } from './constants'
+import { BuyAndSellTokensScreenType } from './index'
+import { getMountedUrl, getValidCurrencyLabel } from './utils'
+
+type TProps = {
+  hidden: boolean
+  setScreenType: Dispatch<BuyAndSellTokensScreenType>
+  account?: IAccountState
+}
+
+export const BuyTokensContent = ({ hidden, account, setScreenType }: TProps) => {
+  const { t } = useTranslation('pages', { keyPrefix: 'buyAndSellTokens.buyTokensContent' })
+  const [neonColor, asphaltColor] = [tailwindTheme.colors.neon.DEFAULT, tailwindTheme.colors.asphalt.DEFAULT]
```

Comment:
Move it outside the component.

It think you can simply create two const assignment.

```suggestion
  const NEON_COLOR = tailwindTheme.colors.neon.DEFAULT
  const ASPHALT_COLOR = tailwindTheme.colors.asphalt.DEFAULT
  
  const BuyTokensContent () => {}
```